core/src/main/scala/org/apache/spark/shuffle/hash/BlockStoreShuffleFetcher.scala

```diff
@@ -17,23 +17,21 @@
 
 package org.apache.spark.shuffle.hash
 
-import scala.collection.mutable.ArrayBuffer
-import scala.collection.mutable.HashMap
+import java.io.InputStream
+
+import scala.collection.mutable.{ArrayBuffer, HashMap}
 import scala.util.{Failure, Success, Try}
 
 import org.apache.spark._
-import org.apache.spark.serializer.Serializer
 import org.apache.spark.shuffle.FetchFailedException
 import org.apache.spark.storage.{BlockId, BlockManagerId, ShuffleBlockFetcherIterator, ShuffleBlockId}
-import org.apache.spark.util.CompletionIterator
 
-private[hash] object BlockStoreShuffleFetcher extends Logging {
-  def fetch[T](
+private[shuffle] object BlockStoreShuffleFetcher extends Logging {
```

Comment:
There was a comment from @JoshRosen here that seems to have gotten lost -- can we leave this as private[hash] for now?  I realize this will need to change later, but no reason to make this change pre-emptively, I don't think (makes the revision history more confusing, to understand why something happened).